contracts/erc20-bridge-sampler/contracts/src/IKyberNetwork.sol

```diff
@@ -21,6 +21,8 @@ pragma solidity ^0.5.9;
 
 interface IKyberNetwork {
```

Comment:
I think we should just rename this contract `IKyberNetworkProxy` and the new one `IKyberNetwork`, if it's not too much trouble.